packages/backend/src/api/controllers/report/filter/getSufficientlySynced.ts

```diff
@@ -1,16 +1,16 @@
 import { ReportRecord } from '../../../../peripherals/database/ReportRepository'
-import { getMaxAssetInBridge } from './getMaxAssetsInBridge'
+import { getNewestAssetsInProject } from './getNewestAssetsInProject'
 import { getSyncedTimestamp } from './getSyncedTimestamp'
 
 export function getSufficientlySynced(reports: ReportRecord[]) {
```

Comment:
Again with the filtering I think that in the future this logic should just be removed. As long as ReportUpdater leaves only relevant records it's fine.